extensions/pos-extension/src/constants/constants.ts

```diff
@@ -0,0 +1,6 @@
+import {composeGid} from '@shopify/admin-graphql-api-utilities';
+
+export const MINIMUM_VERSION_SUPPORTED = '10.13.0';
```

Comment:
Can we leave a comment regarding why this is the minimum supported version? (i.e. 2025.10 not supported in older POS versions)